### PR TITLE
Novel module rename to Document

### PR DESCRIPTION
**Pull request by meesuekim**
- changed name of Novel class and file to  Document
- changed references of Novel and .novel to Document and .document
- fixed \r issues

### CODE REVIEW

File: gender_analysis/analysis/analysis.py [hunk 1]
```diff
@@ -34,14 +34,14 @@ def get_count_words(novel, words):
...
-    >>> from gender_analysis import novel
+    >>> from gender_analysis import document
     >>> summary = "Hester was convicted of adultery. "
     >>> summary += "which made her very sad, and then Arthur was also sad, and everybody was "
     >>> summary += "sad and then Arthur died and it was very sad.  Sadness."
     >>> novel_metadata = {'author': 'Hawthorne, Nathaniel', 'title': 'Scarlet Letter',
     ...                   'corpus_name': 'sample_novels', 'date': '1850',
     ...                   'filename': None, 'text': summary}
-    >>> scarlett = novel.Novel(novel_metadata)
+    >>> scarlett = document.Novel(novel_metadata)
```
Comment (ryaanahmed) [inline gender_analysis/analysis/analysis.py]: this needs to be, e.g., document.Document -- and though the above variable can stay named novel_metadata, it's probably better if we call it document_metadata

**Review by ryaanahmed — CHANGES_REQUESTED** [review]
@meesuekim  --  this is still failing a lot of tests. Looks like in some places, you've changed the import statement for the 'novel' module correctly, but you're still calling "Novel" rather than "Document". I'll point out some places in comments.

File: gender_analysis/analysis/analysis.py [hunk 2]
```diff
@@ -62,11 +62,11 @@ def get_comparative_word_freq(freqs):
     :param freqs: dictionary
     :return: dictionary
 
-    >>> from gender_analysis import novel
+    >>> from gender_analysis import document
     >>> novel_metadata = {'author': 'Hawthorne, Nathaniel', 'title': 'Scarlet Letter',
     ...                   'corpus_name': 'sample_novels', 'date': '1900',
     ...                   'filename': 'hawthorne_scarlet.txt'}
-    >>> scarlet = novel.Novel(novel_metadata)
+    >>> scarlet = document.Novel(novel_metadata)
```
Comment (ryaanahmed) [inline gender_analysis/analysis/analysis.py]: same deal here -- needs to be document.Document

File: gender_analysis/analysis/analysis.py [hunk 3]
```diff
@@ -275,14 +275,14 @@ def dunning_total(m_corpus, f_corpus):
...
-    >>> from gender_analysis import novel
+    >>> from gender_analysis import document
     >>> summary = "Hester was her convicted of adultery. "
     >>> summary += "which made her very sad, and then her Arthur was also sad, and her everybody was "
     >>> summary += "sad and then Arthur her died and it was very sad. her Sadness."
     >>> novel_metadata = {'author': 'Hawthorne, Nathaniel', 'title': 'Scarlet Letter',
     ...                   'corpus_name': 'sample_novels', 'date': '1966',
     ...                   'filename': None, 'text': summary}
-    >>> scarlett = novel.Novel(novel_metadata)
+    >>> scarlett = document.Novel(novel_metadata)
```
Comment (ryaanahmed) [inline gender_analysis/analysis/analysis.py]: and so forth... (more below I think, but you get the picture)

**Review by ryaanahmed — CHANGES_REQUESTED** [review]

**Comment by meesuekim** [comment]
did some more refactoring.
should pass all tests now

**Review by ryaanahmed — APPROVED** [review]
Awesome! Looks good, passes tests. Merging.

**Pull request merged**